### ISSUE TITLE
False alert: TypeScript import type not detected

### ISSUE THREAD

**Issue by sveyret** [depcheck/depcheck#438]
### Bug Description
Since TypeScript 2.9, it is possible to import only types from a module. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#import-types
These imports are not detected and therefore are creating false alerts.

### Code snippets
Where the dependency is used:

```
import('module').Type
```

Where the dependency is listed in package.json:

```
"devDependencies": {
  "module": "x.x.x",
}
```

### Versions
 - `node -v`: any
 - `npm -v`: any
 - `depcheck --version`: 0.8.4

### Extra info
It seems that the PR #429 is already correcting this issue. I will provide another PR because I already did the correction in my system (adding a detector), but do not hesitate and discard it if useless.

**Comment by rumpl** [comment]
There is another PR that was merged for better TS support (#422), it's not perfect yet, needs more work. I'll take a look at yours soon

**Comment by sveyret** [comment]
Thank you @rumpl.
I already use v0.8.4, which, if I am not mistaken, includes PR #422. It does not correct the issue mentioned here.

**Comment by rumpl** [comment]
Closed by #445
